src/components/Charts/EmptyPanel.tsx

```diff
@@ -0,0 +1,10 @@
+/** Renders in place of a chart when there is no data */
+
+import React from 'react';
+import { EmptyPanelContainer } from 'components/Charts/Charts.style';
+
+const EmptyPanel: React.FC = ({ children }) => {
+  return <EmptyPanelContainer>{children}</EmptyPanelContainer>;
```

Comment:
Now that the component doesn't have any other props, we might as well just rename `EmptyPanelContainer` → `EmptyPanel` and export it. 

This is better than exporting an empty wrapper because  with the current implementation, `EmptyPanelContainer` can receive any valid props for a `div`, but `EmptyPanel` can't. For example, with the current code, we can't do this:

```tsx
<EmptyPanel style={{ padding: 30 }}>No data</EmptyPanel>
```

but if we export the style directly, the `style` prop will be picked up correctly.